src/java/org/apache/cassandra/metrics/KeyspaceMetrics.java

```diff
@@ -81,6 +81,18 @@ public class KeyspaceMetrics
     public final Histogram sstablesPerReadHistogram;
     /** Tombstones scanned in queries on this Keyspace */
     public final Histogram tombstoneScannedHistogram;
+    /** Time spent flushing memtables */
+    public final Histogram flushTime;
+    public final Histogram storageAttachedIndexRebuildTime;
+
+    /** Time spent writing SAI */
+    public final Histogram storageAttachedIndexWritingTimeForIndexBuild;
+    public final Histogram storageAttachedIndexWritingTimeForCompaction;
+    public final Histogram storageAttachedIndexWritingTimeForFlush;
+    public final Histogram storageAttachedIndexWritingTimeForOther;
+
+    /** Time spent writing  memtables during compaction */
+    public final Histogram compactionTime;
```

Comment:
CNDB has `CompactionTaskMetrics#duration`

Reply:
that's another thing, I initially thought I had to go there, but these metrics are more low level

CompactionTaskMetrics#duration is more about the overall "task", including etcd and other stuff.
I want this metric because it is comparable with the other `storageAttachedIndexWritingTimeForCompaction`

https://github.com/riptano/cndb/blob/46688e90e2444bde0261a3d499ab6f237a6a306d/cndb-tasks/src/main/java/com/datastax/cndb/compactor/CompactionTaskMetrics.java#L206